internal/config/ociartifact/ociartifact.go

```diff
@@ -98,7 +111,33 @@ func (o *OCIArtifact) Pull(ctx context.Context, img string, opts *PullOptions) (
 
 	// Just supporting one layer here. This can be later enhanced by extending
 	// the PullOptions.
-	layer := layers[0]
+	layer := &layers[0]
+
+	if opts.DatabasePath == "" {
+		opts.DatabasePath = defaultDatabasePath
+	}
+	db := diskv.New(diskv.Options{
```

Comment:
Would it make sense to move this to crio start up? or do we need to have this here for every pull?

Reply:
Probably, I'll check what can be done.

Reply:
I think we can have individual caches per API user. So keeping logically in the same way without using `diskv` any more.